app/uk/gov/hmrc/agentepayeregistration/models/RegistrationRequest.scala

```diff
@@ -20,9 +20,9 @@ import play.api.libs.json.Json
 
 case class RegistrationRequest(agentName: String,
                                contactName: String,
-                               telephoneNumber: Option[String],
+                               phoneNo: Option[String],
```

Comment:
here is ok

Reply:
Ahhhhh ok i see, i'll just change it in the audit event